azurelinuxagent/common/cgroupapi.py

```diff
@@ -278,18 +279,23 @@ def start_extension_command(self, extension_name, command, shell, cwd, env, stdo
         :param stdout: File object to redirect stdout to
         :param stderr: File object to redirect stderr to
         """
+        try:
```

Comment:
I also fixed the issue where the cgroups directories were being deleted... now I re-create them on each command (if they do not exist)